### ISSUE TITLE
Drag'n'drop of clipboarded Event to sibling/child gives unexpected error

### ISSUE THREAD

**Issue by emyoulation** [cdhorn/LinkedView#124]
drag an Event from the Clipboard to a sibling in the Parents and Siblings stack 

```
3617458: ERROR: grampsapp.py: line 174: Unhandled exception
Traceback (most recent call last):
  File "C:\Users\DistrictSupport\AppData\Roaming\gramps\gramps51\plugins\LinkedView\view\frames\frame_base.py", line 225, in drag_data_received
    return self._child_drop_handler(dnd_type, obj_or_handle, data)
  File "C:\Users\DistrictSupport\AppData\Roaming\gramps\gramps51\plugins\LinkedView\view\frames\frame_person.py", line 150, in _child_drop_handler
    self.add_new_person_event(None, event_handle=obj_or_handle)
AttributeError: 'ChildRefFrame' object has no attribute 'add_new_person_event'
```

**Comment by cdhorn** [comment]
Yep the ref drop handler needs support added for that.

**Comment by cdhorn** [comment]
Should be addressed in 233ff37870a95bdb5f45a54a8247b59b6e29d1e3
